### ISSUE TITLE
Pressing `]` next to a closing`]` creates an unbalanced pair

### ISSUE THREAD

**Issue by pedrorgirardi** [nextjournal/codemirror.next-clojure#6]
Pressing `]` inside a vector, right next to `]` doesn't step out of the vector. It inserts an unbalanced `]`, and you can't delete it.

Example:

```clojure
;; '|' is cursor position.
(defn f [|] ...)

;; ... press `]`right before the closing `]`
(defn f [|] ...)

;; ... and you have an unbalanced pair.
(defn f []|] ...)

```

**Comment by mk** [comment]
Hi @pedrorgirardi, thanks for the report. This is a regression, most likely caused by our recent upgrade to CodeMirror 0.14. We'll look into it.

**Comment by mk** [comment]
This has been fixed with b03577c5672472253bcdddb2f8b0a01b12a2dcf0 see https://nextjournal.github.io/codemirror.next-clojure/

**Comment by pedrorgirardi** [comment]
Awesome! Thanks, @mk .
